### PR TITLE
Feature/search component

### PR DESCRIPTION
**Pull request by cameronblandford**
* Fix naming conventions for JSON keys
* Update `.gitignore` to include .DS_Store files and IDE settings
* Add a barebones search component for collections
 * This can be easily changed to add item search in general and within collections, as well as searching by field instead of just a full text search.
* Add a real, working link to the selected item detail view for the schedule layer

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # See http://help.github.com/ignore-files/ for more about ignoring files.
 
+*/.DS_Store
+*.idea/*
```
Comment (birdbrained) [inline .gitignore]: Can you add this to your global .gitignore?

File: app/components/search-collections/component.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+export default Ember.Component.extend({
+    session: Ember.inject.service(),
+    searchText: '',
+    results: null,
+    actions: {
+        search() {
+            // make a call to the collections endpoint
+            const input = this.get('searchText');
+            return Ember.$.get(`${ENV.apiBaseUrl}/api/collections/search/?text__contains=${input}`, (data) => {
+                debugger;
```
Comment (birdbrained) [inline app/components/search-collections/component.js]: Debugger

Comment (cameronblandford) [reply app/components/search-collections/component.js]: womp womp :(

File: .gitignore (hunk 1 above)
Comment (cameronblandford) [reply .gitignore]: I didn't know global .gitignores existed. Sure

**Pull request merged**